test/Model/ConsoleModelTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
Add license and copyright.
See here: https://github.com/zendframework/maintainers/blob/abab4a99f3e6d1fac4d7400232ce30ee0bc95249/template/src/ConfigProvider.php#L2-L6